src/modules/add.js

```diff
@@ -0,0 +1,3 @@
+export default function addTask(arr, obj) {
+  return arr.push(obj);
+}
```

Comment:
Good job with the code but it is more preferable to use an arrow function rather than a regular function.

Reply:
@VSeray I've tried to use an arrow function but the Eslint doesn't allow it.

Reply:
It's good that you tried